### PR TITLE
access key proto

### PR DESCRIPTION
**Pull request by erenming**

### CODE REVIEW

File: core/services/accesskey/accesskey.proto [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        option (google.api.http) = {
+            delete: "/api/credential/access-keys/{id}",
+        };
+    }
+}
+
+message ListAccessKeysRequest {
+    optional GenericEnum.Status status = 1;
+    optional GenericEnum.SubjectType subjectType = 2;
+    optional string subject = 3;
```
Comment (liuhaoyang) [inline core/services/accesskey/accesskey.proto:43]: What does subject mean ?

**Review by liuhaoyang — CHANGES_REQUESTED** [review]

File: core/services/accesskey/accesskey.proto (hunk 1 above)
Comment (erenming) [reply core/services/accesskey/accesskey.proto:43]: subject means the authenticate subject which could be a username, IAM-entity-id, organization-id, etc.

Comment (liuhaoyang) [reply core/services/accesskey/accesskey.proto:43]: It is recommended to use category or kind instead of subject

**Review by liuhaoyang — CHANGES_REQUESTED** [review]

File: core/services/accesskey/accesskey.proto [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    string secretKey = 3;
+    GenericEnum.Status status = 4;
+    GenericEnum.SubjectType subjectType = 5;
+    string subject = 6;
+    string description = 7;
+    google.protobuf.Timestamp createdAt = 8;
+}
+
+message GenericEnum {
+    enum SubjectType {
```
Comment (liuhaoyang) [inline core/services/accesskey/accesskey.proto:91]: In erda, we generally use scope to describe different application levels

**Review by liuhaoyang — CHANGES_REQUESTED** [review]

**Review by liuhaoyang — APPROVED** [review]

**Pull request merged**